cli/commands.js

```diff
@@ -1,62 +1,66 @@
 module.exports = {
-    app: {
+    'app': {
```

Comment:
Why did you add these? They actually make the code a bit harder to read and are unnecessary (except for `spring-controller` where it's needed).